### PR TITLE
FIXED #6314 - [language] Unused language strings in ver. 7.10.8

### LINKED ISSUE #6314: [language] Unused language strings in ver. 7.10.8
['LBL_LOGIN_AS']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/include/language/en_us.lang.php#L3481

['LBL_LOGOUT_AS']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/include/language/en_us.lang.php#L3482

--------------------------------------------------------------------------------

['LBL_FTS_SETTINGS']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L250

['LBL_FTS_SETTINGS_DESC']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L251

['LBL_GLOBAL_TEAM_DESC']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L252

['LBL_GLOBAL_TEAM_SELECT']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L253

['LBL_GLOBAL_TEAM']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L254

['LBL_UPDATE_CHECK_AUTO']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L536

['LBL_UPDATE_CHECK_MANUAL']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L537

['LBL_JJWG_MAPS_ADMIN_DONATE_TITLE']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L796

['LBL_JJWG_MAPS_ADMIN_DONATE_DESC']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L797

['LBL_AOP_OPENING_DAYS']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L854

['LBL_AOP_OPENING_HOURS']

https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L855

['LBL_AOP_CLOSING_HOURS']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Administration/language/en_us.lang.php#L856

--------------------------------------------------------------------------------

['LBL_SEND_BUTTON_KEY']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Meetings/language/en_us.lang.php#L103

--------------------------------------------------------------------------------

['LBL_TRIMSUGARFEEDS']
https://github.com/salesagility/SuiteCRM/tree/v7.10.8/modules/Schedulers/language/en_us.lang.php#L149

### PR DESCRIPTION
**Pull request by Mikeybeer2002**
## Description

Unused language labels have been removed from the code.
## Motivation and Context

https://github.com/salesagility/SuiteCRM/issues/6314
## How To Test This

Check that labels are not being used elsewhere.
## Types of changes

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X ] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [ X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by horus68** [comment]
Just a self ping on language changes!

File: include/language/en_us.lang.php [hunk 1]
```diff
@@ -3494,8 +3494,8 @@
 
 //SecurityGroups
 $app_list_strings['moduleList']['SecurityGroups'] = 'Security Suite Management';
-$app_strings['LBL_LOGIN_AS'] = 'Login as ';
-$app_strings['LBL_LOGOUT_AS'] = 'Logout as ';
+$app_strings['LBL_LOGIN_AS'] = 'Login as';
+$app_strings['LBL_LOGOUT_AS'] = 'Logout as';
```
Comment (likhobory) [inline include/language/en_us.lang.php]: These lines are also unused, why you didn’t remove them?

**Comment by cameronblaikie** [comment]
Assessed :+1:

File: include/language/en_us.lang.php (hunk 1 above)
Comment (cameronblaikie) [reply include/language/en_us.lang.php]: I have taken these out :+1:  https://github.com/salesagility/SuiteCRM/pull/8065

**Comment by horus68** [comment]
@cameronblaikie  @Dillon-Brown LBL_TRIMSUGARFEEDS was removed from modules/Schedulers/language/en_us.lang.php 

but... this label was renamed to LBL_TRIMSUITEFEEDS for modules/Schedulers/_AddJobsHere.php https://github.com/salesagility/SuiteCRM/commit/657fc9ff4d31f0a893c238a56d9d4df20377b4f2

I can't confirm this is still in use but maybe this could cause a problem when you searched for it!
The question is... Is there a prune function that require the label? The labels shows in the Scheduler Job dropdown menu (see https://github.com/salesagility/SuiteCRM/issues/5165 )

**Comment by samus-aran** [comment]
Hey @00MB  :wave: 

Apologies on the delay but we would like to ask if you are able to provide us your shipping details so we can thank you for your hard work and time you’ve dedicated to contributing to SuiteCRM during the 2019 HacktoberFest period! :jack_o_lantern: :ghost: 

[HacktoberFest x SuiteCRM 2019 submission form](https://suitecrm.com/merchandise-data-collection/)

We will be closing the submission in 2 weeks time (Friday 7th February 2020) so we can immediately process your goodies.

If you have any concerns or feedback please reach out to us by emailing [community@suitecrm.com](mailto:community@suitecrm.com).

Would like to thank you again for participating with us on improving the SuiteCRM codebase and we look forward to future contributions from yourself :tada:

**Pull request merged**